docs/src/appendix/cheatcodes/stop_spoof.md

```diff
@@ -4,16 +4,16 @@
 
 Cancels the [`start_spoof`](./start_spoof.md) for the contract at the given address.
```

Comment:
Here, and above there is still `contract_address` instead of CheatTarget

Reply:
Fixed